1-exercises/A-this/exercise-1.js

```diff
@@ -9,18 +9,18 @@
 
 */
 
-
-
 let person = {
-    name: "Adrian",
-    age: 32,
-    currentAddress: "London",
-    celebrateBirthday: /* todo */
+  name: "Adrian",
+  age: 32,
+  currentAddress: "London",
+  celebrateBirthday: function () {
+    return this.age + 1;
+  } /* todo */,
```

Comment:
The question is slightly confusing, what is required is that celebrateBirthday() should increment the age stored in the object by 1 eg:
```celebrateBirthday: function () {
    this.age = this.age + 1;
  },```